components/zcash_protocol/src/local_consensus.rs

```diff
@@ -43,6 +43,37 @@ pub struct LocalNetwork {
     pub z_future: Option<BlockHeight>,
 }
 
+impl LocalNetwork {
+    pub fn new(
+        overwinter: u64,
+        sapling: u64,
+        blossom: u64,
+        heartwood: u64,
+        canopy: u64,
+        nu5: u64,
+    ) -> Self {
+        LocalNetwork {
+            overwinter: Some(BlockHeight::from_u32(overwinter as u32)),
+            sapling: Some(BlockHeight::from_u32(sapling as u32)),
+            blossom: Some(BlockHeight::from_u32(blossom as u32)),
+            heartwood: Some(BlockHeight::from_u32(heartwood as u32)),
+            canopy: Some(BlockHeight::from_u32(canopy as u32)),
+            nu5: Some(BlockHeight::from_u32(nu5 as u32)),
```

Comment:
Constructor is currently broken because it needs to include `nu6` and `z_future` default values behind their respective cfg flags. (Do not expose these in the constructor arguments; they are not necessary for local testing against existing consensus rules, and should be handled separately.)
```suggestion
            nu5: Some(BlockHeight::from_u32(nu5 as u32)),
            #[cfg(zcash_unstable = "nu6")]
            nu6: None,
            #[cfg(zcash_unstable = "zfuture")]
            z_future: None,
```